api_spec.yaml

```diff
@@ -304,10 +304,13 @@ paths:
                       type: string
                     clientId:
                       type: string
+                      $ref: '#/components/schemas/oauth2ClientId'
                     clientSecret:
                       type: string
                     codeVerifier:
```

Comment:
how will the iss, aud be sent to the core?

Reply:
iss and aud, will be  part of `accesstokenPayload` json.